src/test/java/redis/clients/jedis/tests/JedisClusterTest.java

```diff
@@ -579,4 +555,8 @@ private boolean isAnyNodeHandshaking(Jedis node) {
     return false;
   }
 
+  private HostAndPort generateHostAndPort(List<Object> hostInfos) {
```

Comment:
@marcosnils 
If Github isn't lying to me, generateHostAndPort() is here in JedisClusterTest.
https://github.com/charsyam/jedis/blob/feature/using_cluster_slots/src/test/java/redis/clients/jedis/tests/JedisClusterTest.java